content-jumpbox-provisioner/ansible/provision.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: provision stack
+  hosts: localhost
+  connection: local
+
+  vars_files:
+    - vars/{{ environment_type }}-{{ aws_region }}.yaml
+
+  tasks:
```

Comment:
Do we need or want to add a Dyn CNAME for this, like we had with the CoCo jumpoff / old tunnel box?